package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Grafana IoT TwinMaker App Plugin",
   "scripts": {
     "spellcheck": "cspell -c cspell.config.json \"**/*.{ts,tsx,js,go,md,mdx,yml,yaml,json,scss,css}\"",
-    "build": "rm -rf dist && npx grafana-toolkit plugin:build && mage build:backend",
+    "build": "NODE_OPTIONS='--max-old-space-size=8192' grafana-toolkit plugin:build",
```

Comment:
LGTM! This was exactly what Plugins Platform suggested to us when we saw this occurring in other plugins, thanks for updating!

Reply:
just curious: Is this at all related to the issue @iwysiu was dealing with earlier?

Reply:
I don't think so. This is due to some kind of space limitation while the action runs in github. We were also working with the `dev` script which is simpler. 

Also Marcus has replied to Isabella about the issue and they seem to have found a reason for the failure.